plenum/config.py

```diff
@@ -167,11 +159,8 @@
 # Max batch size for 3 phase commit
 Max3PCBatchSize = 100
 # Max time to wait before creating a batch for 3 phase commit
-Max3PCBatchWait = 1
+Max3PCBatchWait = .001
```

Comment:
I think we assumed that timeout is measured is secs before. Do we support milliseconds correctly?